website/templates/base.mako

```diff
@@ -5,11 +5,14 @@
 <head>
     <meta charset="utf-8">
     <title>OSF | ${self.title()}</title>
-    % if settings.GOOGLE_SITE_VERIFICATION:
+    % if settings.GOOGLE_SITE_VERIFICATION and node and node['is_public']:
```

Comment:
are there any non-node, mako-based pages that we still want indexed? can't think of any that haven't been replaced (like osf.io/rr), but that'd be the only worry

Reply:
A quick search yields None, but even if they did most of the metadata uses the `node` data to poulate so on those pages, little of substance is being indexed anyway.

Reply:
There is a `${self.public()}` template function in this file that ostensibly serves the same purpose. I didn't see any use in any template outside of the GA block in this file. Less risk of breaking other things if `node` isn't passed as a templating arg, but more risk of it not working as expected.

Reply:
looks like `${self.public()}` returns a string `in {'public', 'private', 'n/a'}` which we pass to third-party analytics, so probably don't want to reuse for this

could be nice to add `${self.should_amplify_by_search_engine()}` or something that defaults `False`, then move this logic to [`project_base.mako`](https://github.com/CenterForOpenScience/osf.io/blob/HEAD/website/templates/project/project_base.mako), if we want to invest any more attention into mako-template inheritance